README.md

```diff
@@ -32,3 +32,13 @@ This image contains:
 -   An instance of Nginx that proxies Metabase and bypasses auth
 
 It downloads the data provided by the WHO, aggregated by [Johns Hopkins CSSE](https://github.com/CSSEGISandData/COVID-19), transforms it and then loads it into the Postgres instance, ready to be analysed in Metabase
+
+# Licencing
+
+| Element | AGLP | Custom | Link |
+|---------|------|--------|------|
+| Scripting | x  |        |
+| Data    |      |   x    |https://github.com/CSSEGISandData/COVID-19#:~:text=Terms
+| Metabase| x    |        |https://www.metabase.com/license/
+| Nginx   |      |   x    |https://nginx.org/LICENSE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[table-pipes] Missing final pipe in table fence](https://app.codacy.com/manual/jrtapsell/MetabaseAutoCovid/pullRequest?prid=5323405)